src/utils/settingUtils.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+
+const extensionPrefix: string = "azureTerraform";
+
+export function isTerminalSetToCloudShell(): boolean {
```

Comment:
Look like this method is just reading the user settings `terminal`, no logic to check cloudshell. Is this utility really necessary?

Reply:
Sorry for the mistake I made. Have fixed it.